src/grpc/clients/ea_restaurant_java_etl_grpc_client.py

```diff
@@ -18,14 +26,19 @@ def __init__(self, grpc_client):
                 self.channel
             )
         )
+        self.env_config = get_env_config_instance()
+        self.access_token = None
+        self.refresh_token = None
+        self.first_time_login = False
+        self.mongo_uuids = None
 
-    def login_client(self, client_id, client_secret):
+    def login_client_call(self, client_id, client_secret):
```

Comment:
Why not to name it `login_client`? also I think this function should be private. No one else will use it from the oustside

Reply:
Ff i called just `_login_client` it will match other function with same name inside the class.